packages/core/src/models/CustomType/index.ts

```diff
@@ -16,6 +16,7 @@ export const CustomTypeSM = t.exact(
     }),
     t.partial({
       hash: t.string,
+      __status: t.string,
```

Comment:
is this sent to the api or extracted out?

Reply:
extracted out i believe. This info won't be in prismic.io at all